core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -171,7 +171,9 @@ def test_column_order_validation_SegValidator(self):
         # set level according to this test case:
         self.logger.setLevel(logging.ERROR)
         record_list = self.validate('data_seg_wrong_order.txt',
-                                    validateData.SegValidator)
+                                    validateData.SegValidator,
+                                    extra_meta_fields={'reference_genome_id':
+                                                           'hg19'})
```

Comment:
I would like to see a test where you give 'hg18' or some other invalid value (maybe 'sausage' :) and it gives an error.